### PR TITLE
fix: don't cfs filter when cfs is disabled

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5578

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/index.js [hunk 1]
```diff
@@ -215,6 +215,15 @@ export default class IndexRoute extends Route {
...
           val  : params.start_date
         }
+      },
+      {
+        name : 'speakers-call',
+        op   : 'has',
+        val  : {
+          name : 'announcement',
+          op   : 'ne',
+          val  : ''
```
Comment (iamareebjamal) [inline app/routes/index.js:225]: If speaker call is being disabled, announcement should be set to null, not empty string

Comment (maze-runnar) [reply app/routes/index.js:225]: it's giving error `field may not be null`.

Comment (iamareebjamal) [reply app/routes/index.js:225]: Make it nullable in server please

Comment (maze-runnar) [reply app/routes/index.js:225]: `announcement = db.Column(db.Text, nullable=True)` do i have to migrate also after making it nullable.

Comment (iamareebjamal) [reply app/routes/index.js]: Any change in schema means migration

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**